library/core/src/main/java/com/google/android/exoplayer2/audio/SonicAudioProcessor.java

```diff
@@ -129,7 +138,7 @@ public boolean configure(int sampleRateHz, int channelCount, @Encoding int encod
 
   @Override
   public boolean isActive() {
-    return Math.abs(speed - 1f) >= CLOSE_THRESHOLD || Math.abs(pitch - 1f) >= CLOSE_THRESHOLD;
+    return isActive && (Math.abs(speed - 1f) >= CLOSE_THRESHOLD || Math.abs(pitch - 1f) >= CLOSE_THRESHOLD);
```

Comment:
You are right.  This is "passthrough" really.  But there are two use cases for this change.  First is to playback 24bit pcm audio without any dithering so converting to 32bit float is necessary.  Second is to allow FFmpegAudioRenderer to decode any HD audio streams to >16bit audio.

I'll look to make the recommended changes (and below one) next week.  Thanks.